downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -132,12 +132,12 @@ public function main() {
 }
 ```
 
-Since `details` is created as an immutable value, the following check evaluates to true.
+Since `details` is created as an immutable value, the following check evaluates to `true`.
 
 ```ballerina
 details.isReadOnly()
 ```
-### The rest argument can be used to pass arguments for required/defaultable parameters
+### The rest argument to pass arguments for required/defaultable parameters
```

Comment:
This doesn't quite sound right. @MaryamZi thoughts please

Reply:
Verbs or full sentences should not be used in titles.

Reply:
Yeah, I don't think this conveys the idea. Can we please change this to something like 

"Passing arguments for required/defaultable parameters via the rest argument"

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1072.